.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -0,0 +1,50 @@
+param ($GitHub)
+
+switch ($GitHub.event_name)
+{
+    "discussion"
+    {
+        $summary = "Respond to `"$($GitHub.event.discussion.title)`""
+        $description = $Env:DISCUSSION_JIRA_ISSUE_DESCRIPTION
+        $link = $GitHub.event.discussion.html_url
+    }
+    "issues"
+    {
+        $summary = "$($GitHub.event.issue.title) in $($GitHub.repository)"
```

Comment:
This is adding the repo to the title. Perhaps it should rather add the configured component, or no suffix at all. What do you think?

Reply:
Well, the issue component is optional. So it could be the component if available and the repo otherwise.

Reply:
Good idea. I'll actually add this to the other titles too.